api.swagger.yaml

```diff
@@ -437,6 +437,20 @@ paths:
         '403':
           description: 'User is not authorized to view this.'
       description: Returns a count of users who accessed the site on 4 or more days within a given calendar month.
+  /users/metrics/profiles:
+    get:
+      tags:
+        - users
+      responses:
+        '200':
+          description: 'Count of users profile metrics returned.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/user-metrics-profiles-schema'
+        '403':
+          description: 'User is not authorized to view this.'
+      description: Returns a count of users in relation to user profile metrics.
```

Comment:
Not sure this description is clear enough

Reply:
Lindsay gave me these descriptions haha so i will catch up with him :)